#!/usr/bin/env python
PACKAGE = "jsk_2023_09_cook_from_recipe"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("offset_x", int_t, 0, "height number", 1140, 0, 5096)
gen.add("offset_y", int_t, 0, "height number", 480, 0, 5096)
gen.add("width", int_t, 0, "height number", 280, 0, 5096)
gen.add("height", int_t, 0, "height number", 280, 0, 5096)

exit(gen.generate(PACKAGE, "jsk_2023_09_cook_from_recipe", "Crop"))
